evcache-core/src/test/java/com/netflix/evcache/test/SimpleEVCacheTest.java

```diff
@@ -202,6 +214,20 @@ public void testBulk() throws Exception {
         }
     }
 
+    @Test(dependsOnMethods = { "testGetAndTouch" })
+    public void testCompletableFutureBulk() throws Exception {
+        final String[] keys = new String[12];
+        for (int i = 0; i < keys.length; i++) {
+            keys[i] = "key_" + i;
+        }
+        Map<String, String> vals = getAsyncBulk(keys, evCache);
+        assertTrue(!vals.isEmpty());
+        for (int i = 0; i < vals.size(); i++) {
+            String key = "key_" + i;
+            String val = vals.get(key);
```

Comment:
if we aren't validating "val", maybe ignore defining the val variable? I am assuming the intention of vals.get(key) is that the dictionary has a key for every "key" we pass into getAsyncBulk() method - is that correct?